cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,29 @@
+import HomeAndCataloguePageObject
+  from '../support/pages/homeCatalogue.pageObject';
+
+const homePage = new HomeAndCataloguePageObject();
 /// <reference types='cypress' />
 
 describe('', () => {
   before(() => {
-
+    homePage.visit();
   });
 
   it('', () => {
-
+    homePage.clickOnLinkOther('Laptops');
+    cy.get('.hrefch').contains('a', 'Sony vaio i7').click();
+    cy.get('.col-sm-12 > .btn').click();
```

Comment:
After clicking 'Add to cart', you should assert the message in the alert to ensure it matches the expected result as per the task requirements.